deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -529,4 +536,16 @@ private NodeIterator getChildIterator(Resource resc) {
 	private void addAclProperties(Resource depositResc, Model aipModel) {
 		// TODO add access control properties
 	}
+	
+	private void addDescription(ContentObject obj) {
+		File modsFile = new File(getDescriptionDir(), obj.getPid().getUUID() + ".xml");
+		if (!modsFile.exists()) {
+			return;
+		}
+		try(InputStream modsStream = FileUtils.openInputStream(modsFile)) {
+			obj.addDescription(modsStream);
+		} catch(IOException e) {
+			// just eat the exception, or do we want some exception handling here?
```

Comment:
Fix spacing on this try block.  Also, we don't want to eat the exception, we'd want the code calling this method to get the exception and figure out what to do with it